lib/ActivityStream.jsm

```diff
@@ -73,6 +73,7 @@ const PREFS_CONFIG = new Map([
       model_keys: ["nmf_model_animals", "nmf_model_business", "nmf_model_career", "nmf_model_datascience", "nmf_model_design", "nmf_model_education", "nmf_model_entertainment", "nmf_model_environment", "nmf_model_fashion", "nmf_model_finance", "nmf_model_food", "nmf_model_health", "nmf_model_home", "nmf_model_life", "nmf_model_marketing", "nmf_model_politics", "nmf_model_programming", "nmf_model_science", "nmf_model_shopping", "nmf_model_sports", "nmf_model_tech", "nmf_model_travel", "nb_model_animals", "nb_model_books", "nb_model_business", "nb_model_career", "nb_model_datascience", "nb_model_design", "nb_model_economics", "nb_model_education", "nb_model_entertainment", "nb_model_environment", "nb_model_fashion", "nb_model_finance", "nb_model_food", "nb_model_game", "nb_model_health", "nb_model_history", "nb_model_home", "nb_model_life", "nb_model_marketing", "nb_model_military", "nb_model_philosophy", "nb_model_photography", "nb_model_politics", "nb_model_productivity", "nb_model_programming", "nb_model_psychology", "nb_model_science", "nb_model_shopping", "nb_model_society", "nb_model_space", "nb_model_sports", "nb_model_tech", "nb_model_travel", "nb_model_writing"],
       show_spocs: showSpocs(args),
       personalized: true,
+      use_layout: false,
```

Comment:
I dunno, no huge reason, but I guess it's explicit and more reliable (layout/feed can be stuck behind a cache)

But tbh I'm not sure if layout being cached is actually an issue, and I don't know if it being more explicit is helpful at all.

Reply:
It's also follow a pattern already used for spocs.

Reply:
Oh, I guess it also means if people want to opt out, we can just get this pref flipped?

Reply:
If we expose UI for toggling this part of the JSON pref, it will prevent any future default values as firefox sees the whole pref as a custom value.

Reply:
Ah I see, is that why we have a showSponsored pref on top of the show_spocs pref inside the json?

Reply:
@Mardak that's true, but we do need to support the opt-out somehow, and that would be controlled separately from the stuff that the feeds send down, wouldn't it?

If so, this could end up being the opt-out pref, I'd think.

CC @piatra

Reply:
(And the JSON pref that normandy changes would be different)

Reply:
(Assuming we go with a JSON pref, which does seem like a reasonable plan.)

Reply:
I'm fine with dropping the use_layout pref

Reply:
@ScottDowne so I'm assuming that your concern with this code (outside of the pref bit) is simply getting something stood up as a basis for further work.  If that's true, we can certainly sort out the pref store (opt-out vs JSON, etc) separately.  If you want to go that way, just update the patch here, and I'll review it this evening.